atlasdb-cassandra-integration-tests/src/test/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraGetCandidateCellsForSweepingTest.java

```diff
@@ -69,23 +68,4 @@ public void returnCandidateIfTwoCommittedTimestamps() {
                         .build());
     }
 
-    // TODO(nziebart): check that this is ok to fail, and delete
-    @Ignore
-    @Test
-    public void doNotReturnCandidateWithCommitedEmptyValueIfConservative() {
-        new TestDataBuilder().putEmpty(1, 1, 10L).store();
-        assertThat(getAllCandidates(conservativeRequest(PtBytes.EMPTY_BYTE_ARRAY, 40L, 1))).isEmpty();
-    }
-
-    @Test
-    public void returnCandidateWithCommitedEmptyValueIfThorough() {
```

Comment:
Why are we deleting this one?

Reply:
the concept of whether or not to empty values was based on an optimization in the shim which no longer exists. so this test doesn't really make sense anymore

Reply:
cool.